SPECS/python-tensorboard/python-tensorboard.spec

```diff
@@ -0,0 +1,110 @@
+%global pypi_name tensorboard
+%global _description %{expand:
+TensorBoard is a suite of web applications for inspecting and understanding your TensorFlow runs and graphs}
+%define _enable_debug_package 0
+%global debug_package %{nil}
+
+Summary:        TensorBoard is a suite of web applications for inspecting and understanding your TensorFlow runs and graphs
+Name:           python-%{pypi_name}
+Version:        2.11.0
+Release:        1%{?dist}
+License:        ASL 2.0
+Vendor:         Microsoft Corporation
+Distribution:   Mariner
+URL:            https://github.com/tensorflow/tensorboard
+Source0:        https://github.com/tensorflow/tensorboard/archive/refs/tags/%{version}.tar.gz#/%{pypi_name}-%{version}.tar.gz
+Source1:        %{name}-%{version}-cache.tar.gz
+BuildRequires:  python3-setuptools
+BuildRequires:  python3-pip
+BuildRequires:  python3-wheel
+BuildRequires:  python3-six
+BuildRequires:  bazel = 5.3.0
+BuildRequires:  python3-tf-nightly = 2.11.0
+BuildRequires:  gcc
+BuildRequires:  build-essential
+BuildRequires:  protobuf
+BuildRequires:  zlib
+BuildRequires:  python3-virtualenv
+ExclusiveArch:  x86_64
+
+
+%description %{_description}
+
+%package -n python3-%{pypi_name}
+Summary:        %{summary}
+Requires:   python3-absl-py
+Requires:   python3-google-auth-oauthlib = 0.5.3
+Requires:   python3-google-auth = 2.6.6
+Requires:   python3-grpcio 
+Requires:   python3-markdown
+Requires:   python3-numpy
+Requires:   python3-protobuf = 3.17.3
+Requires:   python3-requests = 2.27.1
+Requires:   python3-setuptools
+Requires:   python3-tensorflow-estimator = 2.11.0
+Requires:   python3-tensorflow-io-gcs-filesystem
+Requires:   python3-werkzeug
+Requires:   python3-wheel
+
+
+%description -n python3-%{pypi_name} %{_description}
+
+%package -n python3-%{pypi_name}-data-server
+Summary:        %{summary}
+
+
+%description -n python3-%{pypi_name}-data-server %{_description}
+
+%prep
+%autosetup -p1 -n tensorboard-%{version}
+
+%build
+tar -xf %{SOURCE1} -C /root/
+
+ln -s /usr/bin/python3 /usr/bin/python
+
+#tensorboard-data-server
+pushd tensorboard/data/server/pip_package
+python3 setup.py -q bdist_wheel 
+popd
+mkdir -p pyproject-wheeldir/ && cp tensorboard/data/server/pip_package/dist/*.whl pyproject-wheeldir/
+
+#tensorboard buildt usin bazel
+bazel --batch build //tensorboard/pip_package:build_pip_package
+#cache 
+# ---------
+# steps to create the cache tar. network connection is required to create the cache.
+#----------------------------------
+# bazel clean
+# pushd /root
+# tar -czvf %{name}-%{version}-cache.tar.gz .cache  #creating the cache using the /root/.cache directory
+# popd
+# mv /root/%{name}-%{version}-cache.tar.gz /usr/
```

Comment:
Please move to a script - same ask as what Jon mentioned in another spec.

Reply:
I talked with Nicholas last time about this. Adding these commands to the script file will confuse others as it won't run as intended because it needs Bazel runtime with build requirements from this spec file.
Also, these commands need to run precisely after the Bazel build is successful which is happening in this case.